README.md

```diff
@@ -25,13 +25,13 @@ _Note_:
 
 ### Data Ingress Pipeline Setup (backend)
 
-The next step is to install the latest release of the [Data Ingress Pipeline](https://github.com/Sage-Bionetworks/HTAN-data-pipeline/tree/develop) (backend) as a folder `HTAN-data-pipeline` inside the `HTAN_data_curator` folder and tie it together with this frontend. 
+The next step is to install the latest release of the [Data Ingress Pipeline](https://github.com/Sage-Bionetworks/schematic/tree/main) (backend) as a folder `schematic` inside the `HTAN_data_curator` folder and tie it together with this frontend. 
```

Comment:
should you change to Schematic here?

Reply:
e.g. latest stable release of the [schematic]

Reply:
Right! That makes sense, we don't want users to think "Data Ingress Pipeline" is something different from "Schematic".👍I'll make changes to the README.